examples/build_file_generation/requirements.in

```diff
@@ -1 +1,4 @@
 flask
+sphinx
+sphinxcontrib-serializinghtml
+colorama  # platform-conditional dependency of sphinx
```

Comment:
Why do we need it here? I would assume that locking the requirements on `windows` would work here.

Reply:
Should have cleaned this up.